packages/percy-storybook/src/__tests__/getStories-tests.js

```diff
@@ -1,65 +1,31 @@
 import getStories from '../getStories';
-import { storiesKey } from '../constants';
+// import { storiesKey } from '../constants';
 
 let originalTimeout = 0;
 
 beforeEach(function() {
   originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
-  jasmine.DEFAULT_TIMEOUT_INTERVAL = 12000;
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
```

Comment:
Curious... I assume you wanted to make this bigger because loading a headless browser will likely be a bit slower than `jsdom`?

Reply:
Yes.  15 seconds wasn't quite long enough either.